src/CommandingOfficers/CommanderInfo.java

```diff
@@ -19,9 +20,14 @@ public CommanderInfo(String name)
 
   public abstract Commander create(GameScenario.GameRules rules);
 
+  public void injectUnits(GameReadyModels grms)
+  {
+  }
+
```

Comment:
Why make this part of `CommanderInfo` instead of `Commander`?

Reply:
I wanted to inject the unit type before any `Commander` was instantiated, so all `Commander` constructors see it; otherwise, you could potentially get different behavior depending on CO ordering.
I guess you technically could anyway, but "inject units" is a much more narrow scope than the `Commander` constructor.